### PR TITLE
New package: keepassxc.2.1.4

### PR DESCRIPTION
**Pull request by ibrokemypie**

### CODE REVIEW

File: srcpkgs/keepassxc/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'keepassxc'
+pkgname=keepassxc
+version=2.1.4
+revision=1
+only_for_archs="x86_64 i686"
```
Comment (Gottox) [inline srcpkgs/keepassxc/template]: Are you sure it's only for those arches? Can you provide buildlogs for arm to get an idea what's going wrong there?

File: srcpkgs/keepassxc/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+build_style=cmake
+configure_args="-DWITH_XC_HTTP=ON -DWITH_XC_YUBIKEY=ON -DWITH_XC_AUTOTYPE=ON -DWITH_TESTS=OFF"
+hostmakedepends="make gcc"
+makedepends="libXi-devel libXtst-devel qt5-devel qt5-x11extras-devel qt5-tools-devel libgcrypt-devel zlib-devel libyubikey-devel libykpers-devel"
+depends="qt5"
+short_desc="KeePassXC is a cross-platform port of “Keepass Password Safe”"
+maintainer="ibrokemypie <ibrokemypie@bastardi.net>"
+license="GPL-3"
```
Comment (Gottox) [inline srcpkgs/keepassxc/template]: According to https://github.com/keepassxreboot/keepassxc/blob/develop/COPYING, there are more licenses than only GPL-3 involved.

**Comment by ibrokemypie** [comment]
I have commented out the only_for_arches to show the logs in travis, and all the licenses are now installed correctly.
`CMake Error at /usr/aarch64-linux-gnu/usr/lib64/cmake/Qt5Core/Qt5CoreConfig.cmake:15 (message):
  The imported target "Qt5::Core" references the file
     "/usr/lib/qt5/bin/qmake"
  but this file does not exist.  Possible reasons include:
  * The file was deleted, renamed, or moved to another location.
  * An install or uninstall procedure did not complete successfully.
  * The installation package was faulty and contained
     "/usr/aarch64-linux-gnu/usr/lib64/cmake/Qt5Core/Qt5CoreConfigExtras.cmake"
  but not all the files it references.
Call Stack (most recent call first):
  /usr/aarch64-linux-gnu/usr/lib64/cmake/Qt5Core/Qt5CoreConfigExtras.cmake:6 (_qt5_Core_check_file_exists)
  /usr/aarch64-linux-gnu/usr/lib64/cmake/Qt5Core/Qt5CoreConfig.cmake:146 (include)
  CMakeLists.txt:163 (find_package)
-- Configuring incomplete, errors occurred!`

**Comment by Gottox** [comment]
We'll merge this now and fix crosscompiling later.

**Comment by ebfe** [comment]
This only installs license files. Maybe s/do_install/post_install/ ?

**Pull request merged**